README.md

```diff
@@ -167,6 +163,30 @@ py.test --numprocesses=4 --looponfail --maxfail=1
 pytest -n 4 -f --maxfail=1
 ```
 
+#### How to Execute the Tests?
+
+1. Make sure that you have `leveldb` and `tox` installed
+
+```sh
+brew install leveldb
```

Comment:
This line assumes you are on a Mac. Can you add instructions for other OSs, or at least note that this is Mac-only and a suggestion for people on other OSs to search for their own mechanism to install leveldb?

Reply:
What do you think of breaking the **README** into platforms, on the top of the main one? Thus we will have 4 documents:
1. `README.md` (the main general info)
2. `README-windows.md`
3. `README-osx.md`
4. `README-linux.md`

Can add whatever we know, but then we can iteratively add all of the `TODO`'s there?

Reply:
That sounds reasonable. We can link from any section that needs OS-specific instructions directly into the subsection of the platform readme, like:

---

```md
## Dev Setup

This requires some developer tools before install. See OS-specific instructions for:

- [Linux](docs/README-linux.md#dev-setup)
- [Mac](docs/README-osx.md#dev-setup)
- [Windows](docs/README-windows.md#dev-setup)

Then run these install commands:

'``
pip install -e . -r requirements-dev.txt
'``

```
etc...

Reply:
Agree. Let me do that